core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -532,13 +538,203 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
     stopAndVerifyProduceConsume(producerThread, consumerThread)
   }
 
+  @Test
+  def testAdvertisedListenerUpdate(): Unit = {
```

Comment:
Is it worthwhile having a test case for changing the inter-broker-listener?

Reply:
Sorry for the confusion about inter-broker listeners. Added a test to verify that inter-broker listener cannot be updated.